README.md

```diff
@@ -1 +1,41 @@
-# weather-app
\ No newline at end of file
+# Weather App.
+
+> This project consist on build a webpage where a user can look for the day weather.
+
+
+
+## Built With
+
+- JS
+- Module Pattern
+- Webpack
+- ES6
+- CSS
+
+## Getting Started
+
+- Download the repo con your computer.
```

Comment:
The previous reviewer mentioned this typo...

I trust you can correct this before merging 👍